src/Engine/Spells/CastSpellInfo.cpp

```diff
@@ -166,7 +161,7 @@ void CastSpellInfoHelpers::castSpell() {
             }
         }
 
-        if (pCastSpell->uSpellID < SPELL_BOW_ARROW) {
+        if (pCastSpell->uSpellID <= SPELL_REGULAR_LAST) {
```

Comment:
Just add a bool function to Spells.h that checks what you need and call it here. This way you could also add a doxydoc for the function that people could refer to if it's not clear what's going on in the code.